src/posthog-core.ts

```diff
@@ -870,6 +880,25 @@ export class PostHog {
             this.setPersonPropertiesForFlags(finalSet)
         }
 
+        const beforeSendResult = this.config.before_send(data)
```

Comment:
Not sure why but my gut says it would be better to make this optional and only invoke if set. Thinking ahead to where we _might_ track config settings better and it would be good to know if someone has explicitly implemented it or not

Reply:
yep, on reflection, for support we'd want to know it was configured or not